wger/manager/models.py

```diff
@@ -760,6 +760,12 @@ class WorkoutLog(models.Model):
 
     date = Html5DateField(verbose_name=_('Date'))
 
+    session = models.ForeignKey('WorkoutSession',
+                                null=True,
+                                blank=True,
```

Comment:
Why do you set null to True?

Reply:
A log is referenced to a session. Not setting null to True will require providing a default value for the field for a non-existence session.